config/moved_config.go

```diff
@@ -96,6 +96,7 @@ func (cfg *Config) Validate() error {
 	return cfg.validateService()
 }
 
+// Deprecated: [v0.55.0] Use mustValidateService instead.
```

Comment:
This is private, why do you need deprecation?

Reply:
I didn't notice that. how to indicate it is deprecated for private function? could your give me a hint?

Reply:
just break it/change it... nobody except us is using that func.

Reply:
I've already removed the comment.